src/store/config/selectors.ts

```diff
@@ -25,3 +26,25 @@ export const getFirestoreConfigResult = createSelector(
   getConfigResult,
   result => map(result, config => config.firestore)
 );
+
+export function useProjectId() {
+  const projectId = useSelector(getProjectIdResult);
```

Comment:
This is a good use of useSelector since we don't need to test this one. Good job! Let's still take care before doing `useProjectId` anywhere else though.

Reply:
Ack